### ISSUE TITLE
Improper output formatting

### ISSUE THREAD

**Issue by P4N74** [zachriggle/pwndbg#19]
From the recent merge @br0ns, it seems to clear the home and end key issue in prompt, but the coloured output seems to come out weird. Upon looking into code of `color.py` which has
```
SOH            = "\x01"
STX            = "\x02"
NORMAL         = SOH + "\x1b[0m" + STX
BLACK          = SOH + "\x1b[30m" + STX
RED            = SOH + "\x1b[31m" + STX
GREEN          = SOH + "\x1b[32m" + STX
YELLOW         = SOH + "\x1b[33m" + STX
BLUE           = SOH + "\x1b[34m" + STX
PURPLE         = SOH + "\x1b[35m" + STX
CYAN           = SOH + "\x1b[36m" + STX
GREY = GRAY    = SOH + "\x1b[90m" + STX
BOLD           = SOH + "\x1b[1m" + STX
UNDERLINE      = SOH + "\x1b[4m" + STX
```
[Here](http://imgur.com/KI1HpN0) is the link to image of output.
instead using it as it was before:
```
NORMAL         = "\x1b[0m"
BLACK          = "\x1b[30m"
RED            = "\x1b[31m"
GREEN          = "\x1b[32m"
YELLOW         = "\x1b[33m"
BLUE           = "\x1b[34m"
PURPLE         = "\x1b[35m"
CYAN           = "\x1b[36m"
GREY = GRAY    = "\x1b[90m"
BOLD           = "\x1b[1m"
UNDERLINE      = "\x1b[4m"
```
and adding following line to `__init__.py`:
```
prompt = "pwn> "
prompt = "\x01" + prompt + "\x02"    # SOH + prompt + STX
prompt = pwndbg.color.red(prompt)
prompt = pwndbg.color.bold(prompt)
```
Should fix the error of home and end key in prompt while not interfering with the coloured output due to other functions.

Above changes solves both issues correctly on my system and if you think it is valid to do so, I can submit a pull request.

**Comment by zachriggle** [comment]
Agreed that this is cleaner.  Fixed in c2f0a69.

**Comment by zachriggle** [comment]
FWIW, I did not see the odd symbols that you do -- but I like this method better.
